src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -598,9 +598,16 @@ static public boolean setAllVirtual(boolean b) {
   /**
    * start
    * @param b
+   * @throws IOException 
    */
-  static void setAutoStart(boolean b) {
+  public void setAutoStart(boolean b) throws IOException {
+    log.error("setAutoStart {}", b);

```

Comment:
Shouldn't be an error, maybe don't log at all

Reply:
changed it to debug